src/pages/workspace/WorkspaceOverviewPage.js

```diff
@@ -106,7 +107,7 @@ function WorkspaceOverviewPage({policy, currencyList, route}) {
                         disabled={readOnly}
                         disabledStyle={styles.cursorDefault}
                     />
-                    <OfflineWithFeedback pendingAction={lodashGet(policy, 'pendingFields.generalSettings')}>
+                    <OfflineWithFeedback pendingAction={pendingAction}>
```

Comment:
I think it makes more sense to wrap each `MenuItemWithTopDescription` with a separate `OfflineWithFeedback`, otherwise we grey out all the items even if only one has changed.

Reply:
@jjcoffee The problem is that the `name` and `currency` are under `generalSettings` and grouped together while description has different `API` endpoint. This means that if we have separate `OfflineWithFeedback`, we would see that upon editing the name, the currency would also get greyed out while the description in between them would not. I think the solution to this is to break up `name` and `currency` into separate endpoints, but I think that could be out of scope for the issue. Please let me know what you think!

Reply:
Commented on Slack too. I think it makes sense to separate it out as it's as separate API command. Once a followup deals with it, the other two fields will update separately too.